scripts/deploy_generator.ts

```diff
@@ -86,49 +89,49 @@ async function uploadAndInitVesting(terra: LCDClient, wallet: any) {
                 owner: network.multisigAddress,
                 token_addr: network.tokenAddress,
             },
+            VESTING_LABEL
         )
-        network.vestingAddress = resp.shift()
-        console.log(`Vesting Contract Address: ${network.vestingAddress}`)
+        // @ts-ignore
+        network.vestingAddress = resp.shift().shift()
+        console.log(`Address Vesting Contract: ${network.vestingAddress}`)
         writeArtifact(network, terra.config.chainID)
     }
 }
 
 async function uploadAndInitGenerator(terra: LCDClient, wallet: any) {
     let network = readArtifact(terra.config.chainID)
 
-    checkParams(network, ["factoryAddress", "vestingAddress", "assemblyAddress", "whitelistCodeID"])
-
     if (!network.generatorAddress) {
-        console.log('Deploy the Generator...')
+        console.log('Deploying Generator...')
 
         let resp = await deployContract(
             terra,
             wallet,
             network.multisigAddress,
             join(ARTIFACTS_PATH, 'astroport_generator.wasm'),
             {
-                owner: wallet.key.accAddress,
+                owner: network.multisigAddress,
```

Comment:
Revert this line. Usually, we need to have owner rights to setup generator. After setup, you need to create change ownership proposal.